.github/workflows/build.yml

```diff
@@ -263,7 +263,7 @@ jobs:
   #
   windows:
     name: Windows
-    runs-on: windows-2016
+    runs-on: windows-2019
```

Comment:
Take this commit out of your pull-request - a PR should really just focus on one thing at a time. Here is an example of why: the windows change you've made isn't enough to fix our windows problem. We do need to use a later version of windows, but we also need to use a later version of Visual Studio, and a couple of other things will need to be changed too to make it all work. Nobody has figured out exactly how to do this yet - if you want to work on it, you can do it in a separate PR. For now, leave it out of this PR.

Reply:
Sure. Will keep it in mind.